care/users/models.py

```
@@ -488,3 +493,84 @@ def check_user_has_flag(cls, user_id: int, flag_name: FlagName) -> bool:
+    user_agent = models.CharField(max_length=256, blank=True)
+    ip_address = models.GenericIPAddressField(blank=True, null=True)
+    is_used = models.BooleanField(default=False)
+
+    class Meta:
+        verbose_name = "Password Reset Token"
+        verbose_name_plural = "Password Reset Tokens"
+
+    def __str__(self):
+        return f"Password reset token for user {self.user}"
+
+    def save(self, *args, **kwargs):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)